### ISSUE TITLE
[Assistive tech and device support] Focus state isn't visible. (11.09.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#37069]
#### VFS product name
eGain

#### Point of Contact/Reviewers
Sarah Koomson (@Saraharaba8) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Focus state isn't visible.

### Issue Details
Visual indicator around "Start Chat" is nearly invisible. Focus style needs to be overriden with a more accessible one.

### VA.gov Experience Standard
[Category Number 11, Issue Number 09](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.7_AA

---

### Platform Recommendation
All visual focus indicators need to have a color contrast ratio of at least 3:1 against adjacent colors. Default indicators provided by broswers may end up failing, depending on colors associated with your page. Colors of your page are clashing with the default focus indicators making them really difficult to see. Please override these styles. 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.
